{"name": "test_update_package[data0]", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable", "trace": "@pytest.fixture(name='beginandend')\n    def beginToend():\n        lu = LoggerUtil()\n        logger = lu.get_logger()\n        logger.info(\"----------测试用例执行开始----------\")\n        global driver\n>       driver = webdriver.firefox()\nE       TypeError: 'module' object is not callable\n\nTestCase\\conftest.py:20: TypeError"}, "attachments": [{"name": "log", "source": "b797a523-e993-4272-a637-fef7bc91da26-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "['test01', 1, '测试', '描述', '角标', 50, None]"}], "start": 1692770008459, "stop": 1692770008459, "uuid": "1364e099-5bc8-41ce-b2be-48953738de1a", "historyId": "86fd89cac6e70c4da7d3c13e0fa56df0", "testCaseId": "6d51739d3f2cbeea83ddeb68d559af44", "fullName": "TestCase.test_table.Testtable#test_update_package", "labels": [{"name": "feature", "value": "测试360搜索"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beginandend')"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_table"}, {"name": "subSuite", "value": "Testtable"}, {"name": "host", "value": "DESKTOP-14KMC4A"}, {"name": "thread", "value": "30528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_table"}]}